packages/ffe-form/less/radio-switch.less

```diff
@@ -28,6 +28,10 @@
     @media screen and (min-width: @breakpoint-sm) {
         margin-bottom: 0;
     }
+
+    & + input + .ffe-radio-switch {
```

Comment:
This change is non-breaking I take it (from the commit message)?

Reply:
Yep that's right. Let me explain: 

In the previous implementation, two radio switch elements (there are typically two of them) had to be separated by inline styles. That was the way this was solved in the existing React implementation at least. This approach solves this issue by a clever CSS selector. 

To make it a bit easier to understand what I'm doing - here's the typical DOM structure of a pair of radio switches:

```html
<input id="yes" type="radio" class="ffe-radio-input" />
<label for="yes" class="ffe-radio-switch">Yes</label>

<input id="no" type="radio" class="ffe-radio-input" />
<label for="no class="ffe-radio-switch">No</label>
```

I guess I could have used the class instead, but in reality that doesn't matter much.